tests/scientific/test_neuromodulation.py

```diff
@@ -24,6 +25,8 @@ def test_neuromodulation_sims_neuron():
     npt.assert_allclose(timestamps, obtained_timestamps)
 
 
+@pytest.mark.skip(
+    reason="Disable until new CoreNEURON synapse replay is deployed in the neurodamus-neocortex")
```

Comment:
For CoreNEURON replay with netcons, we need to replace `VecStim.mod` with `vecevent.mod` to be compatible with CoreNEURON, which requires https://github.com/BlueBrain/neurodamus-models/pull/8.  
Some part of neurodamus unit tests (scientific related) load the `unstable neocortex` module instead of building on the flight during testing. So I disable those unit tests requiring synapse replays in CoreNEURON at the moment and will enable them after the new vecevent is deployed on bb5.

Reply:
With the latest deployment of `neurodamus-models-2.2.0` which uses `vecevent.mod`, these tests are not skipped any more.